clang-tools-extra/test/clang-tidy/checkers/abseil/string-find-startswith.cpp

```diff
@@ -22,6 +22,21 @@ struct basic_string {
 typedef basic_string<char> string;
 typedef basic_string<wchar_t> wstring;
 
+template <typename C>
```

Comment:
Actually string and string_view is defined already in header test/clang-tidy/checkers/Inputs/Headers/string.

Just add `-- -isystem %clang_tidy_headers` like tests for readability-string-compare does, and simply include string.